src/loader/BlendLoader.py

```diff
@@ -9,8 +10,29 @@
 
 class BlendLoader(LoaderInterface):
     """
-    Loads entities (everything that can be stored in a .blend file's folders, see Blender's documentation for
-    bpy.types.ID for more info) that match a name pattern from a specified .blend file's section/datablock.
+    This class provides functionality to load entities from a .blend file. A .blend file is a 
+    blender generated  data file that wraps project resources into sections/datablocks. Resources can be
+    loaded individually by name pattern matching or entire datablocks to entire project. For more
+    information about a datablock see Blender's documentation for bpy.types.ID. 
+
+    Sections/Datablocks in a .blend File
+    +-------------------+
+    |    Blend File     |
+    +-------------------+
+    |    Collections    |
+    |    Object         |
+    |    Mesh           |
+    |    Text           |
+    |    Scene          |
+    |    World          |
+    |    Workspace      |
+    |    Curve          |
+    |    Camera         |
+    |    Light          |
+    |    Material       |
+    |    Texture        |
+    |    ......         |
+    +-------------------+
```

Comment:
This is really nice. However, it does not work with the docu: https://dlr-rm.github.io/BlenderProc/index.html

So make sure that it also works with that. You can use the `docs/generate.sh` to locally generate the docu.

Reply:
It should be styled as [Available:](https://github.com/DLR-RM/BlenderProc/blob/master/src/object/PhysicsPositioning.py#L54)